language/move-prover/tools/check_pr.sh

```diff
@@ -120,6 +124,13 @@ if [ ! -z "$GEN_ARTIFACTS" ]; then
       cd $dir
       cargo run
     )
+    if [[  $BUILD_EXPERIMENTAL = "$dir"  ]]; then
```

Comment:
```suggestion
    if [[  $BUILD_EXPERIMENTAL == "$dir"  ]]; then
```

Not an expert in bash though...

Reply:
Good spot! I got my `[[` and `[` commands mixed up... should be fixed now